### PR TITLE
Remove old jelly schema and usages

### PR DESCRIPTION
**Pull request by sladyn98**
This PR deals with the removal of the old schema considering the generation of the new Schema https://github.com/jenkinsci/configuration-as-code-plugin/pull/980
Fixes https://github.com/jenkinsci/configuration-as-code-plugin/issues/1057

### Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [x ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [ x] Ensure that the pull request title represents the desired changelog entry
- [ x] Please describe what you did
- [ ] Link to relevant issues in GitHub or in [Jenkins JIRA](https://issues.jenkins-ci.org)
- [x ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### CODE REVIEW

**Comment by sladyn98** [comment]
cc @timja

File: plugin/src/main/resources/io/jenkins/plugins/casc/ConfigurationAsCode/index.jelly [hunk 1]
```diff
@@ -54,7 +54,7 @@
     <h2>${%Reference}</h2>
     <dt>
       <dl><a href="reference">${%Documentation}</a></dl>
-      <dl><a href="v2Schema">${%JSON schema}</a></dl>
+      <dl><a href="Schema">${%JSON schema}</a></dl>
```
Comment (timja) [inline plugin/src/main/resources/io/jenkins/plugins/casc/ConfigurationAsCode/index.jelly]: does `schema` work?

File: plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java [hunk 2]
```diff
@@ -405,7 +405,7 @@ public void doExport(StaplerRequest req, StaplerResponse res) throws Exception {
      * Export JSONSchema to URL
      * @throws Exception
      */
-    public void dov2Schema(StaplerRequest req, StaplerResponse res) throws Exception {
+    public void doSchema(StaplerRequest req, StaplerResponse res) throws Exception {
```
Comment (timja) [inline plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java]: do the docs need updating?

Comment (sladyn98) [reply plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java]: Yeah they would :)

File: plugin/src/main/resources/io/jenkins/plugins/casc/ConfigurationAsCode/index.jelly (hunk 1 above)
Comment (sladyn98) [reply plugin/src/main/resources/io/jenkins/plugins/casc/ConfigurationAsCode/index.jelly]: I think it doesn't matter it still converts to small case

Comment (timja) [reply plugin/src/main/resources/io/jenkins/plugins/casc/ConfigurationAsCode/index.jelly]: should be lower then
```suggestion
      <dl><a href="schema">${%JSON schema}</a></dl>
```

**Comment by sladyn98** [comment]
@timja the Build is failing ...? IO Exception

**Comment by sladyn98** [comment]
cc @oleg-nenashev Needs review .Thanks

**Pull request merged**